bin/picca_compute_fvoigt_hcd.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+import argparse
+
+import numpy as np
+from scipy.constants import speed_of_light
+from scipy.special import wofz
+
+from picca import constants
+
+
+def voigt_profile(x, sigma=1, gamma=1):
+    return np.real(wofz((x + 1j*gamma) / (sigma * np.sqrt(2))))
+
+
+def get_voigt_profile_wave(wave, z, N_hi):
+    """Compute voigt profile at input redshift and column density
+    on an observed wavelength grid
+
+    Parameters
+    ----------
+    wave : array
+        Observed wavelength
+    z : float
+        Redshift
+    N_hi : float
+        Log10 column density
+
+    Returns
+    -------
+    array
+        Flux corresponding to a voigt profile
+    """
+    # wave = lambda in A and N_HI in log10 and 10**N_hi in cm^-2
+    wave_rf = wave / (1 + z)
+
+    e = 1.6021e-19  # C
```

Comment:
These should be defined as constants outside of the function. Consider importing them from `py/picca/delta_extraction/masks/dla_mask.py` or at least change the variable names to have a similar naming convention